pyaedt/modeler/multiparts.py

```diff
@@ -381,16 +381,17 @@ def offset(self, o):
 
     @aedt_exception_handler
     def position_in_app(self, app):
-        """Set up design variables and values to enable motion for the multi-part 3D component in the application.
+        """Set up design variables and values to enable motion for the multi-part 3D component.
 
         Parameters
         ----------
-        app : pyaedt.Hfss, required
-            HFSS pplication instance.
+        app: pyaedt.Hfss
+            HFSS application instance.
 
         Returns
         -------
-        Coordinate system.
+        str
+            Name of the coordinate system.
```

Comment:
```suggestion
        :class:`pyaedt.modeler.Modeler.CoordinateSystem`
```

I think this is returning a coordinate system object, not a string.